share/spack/pip/requirements.txt

```diff
@@ -0,0 +1,5 @@
+build
```

Comment:
Does the pip installed version of spack still use bootstrapping? One request was to vendor clingo via pip. Is that accounted for in this PR? If not is there a compelling reason to NOT vendor via pip?

Reply:
FYI, this does work as anticipated. "spack -b find" does not report a bootstrapped clingo, it uses the one from pip.